app/src/assets/labware/getLabware.ts

```diff
@@ -7,50 +6,50 @@ import type {
 
 // require all definitions in the labware/definitions/1 directory
 // require.context is webpack-specific method
-const labwareSchemaV1DefsContext = (require: any).context(
+const labwareSchemaV1DefsContext = (require as any).context(
```

Comment:
Unnecessary cast thanks to webpack-env defs

```suggestion
const labwareSchemaV1DefsContext = require.context(
```